langchain4j-core/src/main/java/dev/langchain4j/model/image/ImageModel.java

```diff
@@ -0,0 +1,9 @@
+package dev.langchain4j.model.image;
+
+import dev.langchain4j.data.image.Image;
+import dev.langchain4j.model.output.Response;
+import java.util.List;
+
+public interface ImageModel {
+  Response<List<Image>> generate(String prompt);
```

Comment:
@jdubois please check this as well

Reply:
Yes, I already have this, thanks!

Reply:
I haven't pushed yet because of the base64 part, it will be there soon

Reply:
@jdubois this is to your comment
> This is fixing my 2 biggest issues with this API (not limiting to just 1 image, and being able to get the image content and not just the URL)


I am not convinced we should return a List, I would start from a single Image and later on we can add another mothod to return more (if required)

Reply:
We could do both....
Also we need to add a new parameter here to support both base64 and URL, I'll give it a try in my PR

Reply:
both works, although a bit premature. In any case my main point here is to avoid forcing user to go `.get(0)` all the time

Reply:
Argh, I couldn't make it work and just read the doc: ` Dall-e-3 models only support a value of 1.`!! So that would only be for DALL E 2, so not really worth it.
I'll just return one value, then.

Reply:
guys, I don't really won't to throw my support of multiple images, instead I would have two methods:
```
public interface ImageModel {
  Response<Image> generate(String prompt);
  default Response<List<Image>> generateMultiple(String prompt) {
    throw new IllegalArgumentException("Operation is not supported");
  }
}
```
like it's non blocking, what do you think?

Reply:
Why do you want to have multiple images at once? It is currently supported only with outdated dalle2 model...

Reply:
I speak as user, then sure I want to have image variants to my prompt to choose from and technically - the response from DALL·E contains always a list, I suppose DALL·E 3 will support it, this is just matter of time

Reply:
Ok, how about `Response<List<Image>> generate(String prompt, int n)`?

Reply:
sure, I like this!